### PR TITLE
Issue 14344: Contact import is relocated to a dedicated place

### LINKED ISSUE #14344: Moving the function "Your old ActivityPub/GNU social account"
### Is the feature request related to a problem? Please describe.

Friendica has a function that can be used to import contacts without a CSV file. The handle of the remote account is specified for this purpose. Friendica contacts the account and imports the contacts to the "new" account (online). 

This function is currently so well hidden that even long-time Friendica users are unaware of it. 
https://.../settings/connectors > General social media settings > Your old ActivityPub/GNU social account

I would expect to find this function in the easily accessible menu: https://.../settings > "Import contacts".

### Describe the feature you'd like

To make online importing easier, it would be an advantage if the online import function for contacts were moved to https://.../settings > "Import contacts" or at least a deep link were set.

grafik

### PR DESCRIPTION
**Pull request by annando**
Fixes #14344

### CODE REVIEW

File: view/templates/settings/contactimport.tpl
Comment (MrPetovan) [inline view/templates/settings/contactimport.tpl:1]: What happens if you use the Frio template as a base template viewing it in vier? I'd like us to stop writing new templates for vier as it is deprecated.

Comment (annando) [reply view/templates/settings/contactimport.tpl:1]: Both themes use different class names.

Comment (MrPetovan) [reply view/templates/settings/contactimport.tpl:1]: Sure, but I don't think the vier class names are relevant anymore since we barely maintain it.

Comment (annando) [reply view/templates/settings/contactimport.tpl:1]: I think that that new page should look like a page under "vier" normally looks like.

**Review by MrPetovan — APPROVED** [review]

File: view/templates/settings/contactimport.tpl
Comment (MrPetovan) [reply view/templates/settings/contactimport.tpl:1]: I'm not going to hold this PR open because of this, but these are more files we need to maintain for a theme on its way out. 😕

**Comment by foss-** [comment]
Only seeing this, after the PR was merged.

Not a fan of `Import Contacts` being a top level entry in the settings menu. Since `Import Contacts` will not be used often, wouldn't this be better suited as menu item in the `Account` section?

**Comment by annando** [comment]
It is an action, not a setting. And thus I think it doesn't belong to the account settings. But one could argue that it belongs to the contact actions.

**Comment by foss-** [comment]
> It is an action, not a setting. And thus I think it doesn't belong to the account settings. But one could argue that it belongs to the contact actions.

The settings and the settings nomenclature is somewhat inconsistent I think.

- the differentiation between action and setting is a technical aspect, which I don't think are categories users think in. There are some other actions in the settings as well. `Import Contacts` is still listed under settings, as the top level is also reached via the `Settings` menu option.
- moving `Import contacts` to the Contacts page is possible, but the same reasoning would apply: this action is rarely used  (e.g. compared to adding a single new contact) and I don't think it should clutter prominent space.
- as for `Account Settings`: not a fan of calling every sub setting `XYZ Setting` - calling it `XYZ` is sufficient. Example: instead of `Security and Privacy Settings` just call that sub-item `Security and Privacy` since users are in the settings already.

EDIT: maybe I misunderstood what you were suggesting @annando: If you were suggesting to add `Import Contacts` as link to the `Find People` box in the contacts page, I would very much support that idea. I still believe, that it should be removed from the top level settings options.

**Comment by annando** [comment]
I think that especially new users don't find this action easily. So it should have got a higher visibility.

**Comment by foss-** [comment]
That is correct. That is why I thought the account settings were the appropriate location for the feature. When users create a new account or migrate an existing fediverse account, the account settings are what they will be looking at for migration and export / import settings.

**Pull request merged**
